models/ccsr/final/ccsr__long_procedure_category.sql

```diff
@@ -3,8 +3,40 @@
    )
 }}
 
+{% if target.type == 'fabric' %}
+
+with "procedure" as (
```

Comment:
Could we get away with renaming CTEs like this to "procedures" instead of "procedure" since that's a reserved word? Might make the code more simple.

Reply:
Great idea. I will update.

Reply:
Updated